apps/design/shared/src/layout.ts

```diff
@@ -682,10 +747,50 @@ function CandidateContest({
   const optionTextAlign =
     bubblePosition === BallotTargetMarkPosition.Left ? 'left' : 'right';
 
-  const optionRowHeight = 2;
```

Comment:
The next bit of diff adds text wrapping for candidate names, dynamically computing the height needed for each contest option.